senza/cli.py

```diff
@@ -982,7 +982,7 @@ def status(stack_ref, region, output, w, watch):
                         except:
                             answers = []
                         for answer in answers:
-                            if answer.target.to_text().startswith('{}-'.format(stack.StackName)):
+                            if str(answer.target.to_text()).startswith('{}-'.format(stack.StackName)):
```

Comment:
The proper way to do this would be to use `.decode` if `to_text()` returns `bytes` or something with a `decode` method (the second feels more pythonic to me).